tests/acceptance/cli_test.py

```diff
@@ -536,6 +536,16 @@ def test_up_external_networks(self):
         assert sorted(list(container.get('NetworkSettings.Networks'))) == sorted(network_names)
 
     @v2_only()
+    def test_networks_schema_validation(self):
+        filename = 'invalid-network-fields.yml'
+
+        self.base_dir = 'tests/fixtures/networks'
+        self._project = get_project(self.base_dir, [filename])
+
+        result = self.dispatch(['-f', filename, 'up', '-d'], returncode=1)
+        assert 'Additional properties are not allowed' in result.stderr
```

Comment:
We test the rest of this validation in `tests/unit/config/config_test.py`, I think we should move this test there as well.  You can inline the `invalid-network-fields.yml` config in the test for the unit test as well.